docs/content/en/docs/getting-started/production-environment/nutanix-getstarted.md

```diff
@@ -0,0 +1,216 @@
+Make sure you use single quotes around the values so that your shell does not interpret the values:
+   
+   ```bash
+   export EKSA_NUTANIX_USERNAME='billy'
+   export EKSA_NUTANIX_PASSWORD='t0p$ecret'
+   ```
+
+   {{% alert title="Note" color="primary" %}}
+   If you have a username in the form of `domain_name/user_name`, you must specify it as `user_name@domain_name` to
+   avoid errors in cluster creation. For example, `nutanix.local/admin` should be specified as `admin@nutanix.local`.
+   {{% /alert %}}
+
+1. Set License Environment Variable
+
+   Add a license to any cluster for which you want to receive paid support. If you are creating a licensed cluster, set and export the license variable (see [License cluster]({{< relref "/docs/tasks/cluster/cluster-license" >}}) if you are licensing an existing cluster):
+
+   ```bash
+   export EKSA_LICENSE='my-license-here'
+   ```
+
+   After you have created your `eksa-mgmt-cluster.yaml` and set your credential environment variables, you will be ready to create the cluster.
+
+
+1. Configure Curated Packages
+
+   The Amazon EKS Anywhere Curated Packages are only available to customers with the Amazon EKS Anywhere Enterprise Subscription. To request a free trial, talk to your Amazon representative or connect with one [here](https://aws.amazon.com/contact-us/sales-support-eks/). Cluster creation will succeed if authentication is not set up, but some warnings may be generated.  Detailed package configurations can be found [here]({{< relref "../../tasks/packages" >}}).
+
+   If you are going to use packages, set up authentication. These credentials should have [limited capabilities]({{< relref "../../tasks/packages/#setup-authentication-to-use-curated-packages" >}}):
+   ```bash
+   export EKSA_AWS_ACCESS_KEY_ID="your*access*id"
+   export EKSA_AWS_SECRET_ACCESS_KEY="your*secret*key"  
+   ```
+     
+1. Create cluster
+
+   ```bash
+   eksctl anywhere create cluster \
+      # --install-packages packages.yaml \ # uncomment to install curated packages at cluster creation
+      -f eksa-mgmt-cluster.yaml
+   ```
+
+1. Once the cluster is created you can use it with the generated `KUBECONFIG` file in your local directory:
+
+   ```bash
+   export KUBECONFIG=${PWD}/${CLUSTER_NAME}/${CLUSTER_NAME}-eks-a-cluster.kubeconfig
+   ```
+1. Check the cluster nodes:
+
+   To check that the cluster completed, list the machines to see the control plane, etcd, and worker nodes:
+
+   ```bash
+   kubectl get machines -A
+   ```
+
+   Example command output
+   ```
+   NAMESPACE   NAME                PROVIDERID        PHASE    VERSION
+   eksa-system mgmt-b2xyz          nutanix:/xxxxx    Running  v1.21.2-eks-1-21-5
+   eksa-system mgmt-etcd-r9b42     nutanix:/xxxxx    Running  
+   eksa-system mgmt-md-8-6xr-rnr   nutanix:/xxxxx    Running  v1.21.2-eks-1-21-5
+   ...
+   ```
+
+   The etcd machine doesn't show the Kubernetes version because it doesn't run the kubelet service.
```

Comment:
Etcd isn't supported yet. Let's remove all external etcd references for now

Reply:
Good catch. Done.